scripts/build.sh

```diff
@@ -14,16 +15,27 @@ build_local_and_docker() {
    printf '%s' "Building $title Locally...  "
    cwd=$PWD
    cd $folder
-   go build -mod=readonly -trimpath -o $BUILDDIR ./... 2>&1 | grep -v -E "deprecated|keychain";
-   local_build_succeeded=$?
+   GOBIN=$BUILDDIR go install -mod=readonly -trimpath ./... 2>&1 | grep -v -E "deprecated|keychain" | true
```

Comment:
surprised `build` vs `install` is so different!

Reply:
go build is inconsistent